chart/templates/logs-persistent-volume-claim.yaml

```diff
@@ -46,7 +46,7 @@ spec:
   {{- if (eq "-" .Values.logs.persistence.storageClassName) }}
   storageClassName: ""
   {{- else }}
-  storageClassName: "{{ .Values.logs.persistence.storageClassName }}"
+  storageClassName: "{{ tpl .Values.logs.persistence.storageClassName . }}"
```

Comment:
Can you make it a general solution by also addressing the usage in `dags-persistent-volume-claim.yaml` too instead of solving just for logs

Reply:
Sure @amoghrajesh @hussein-awala Added tpl function to all PVCs